packages/diracx-web-components/components/shared/FilterForm.tsx

```diff
@@ -14,6 +14,10 @@ import {
 } from "@mui/material";
 import { Filter } from "@/types/Filter";
 import { Column } from "@/types/Column";
+import { DateTimePicker, LocalizationProvider } from "@mui/x-date-pickers";
```

Comment:
Did you check whether there was any kind of limitation with the `x-date-pickers`?

Reply:
I did check on the documentation and couldn't find anything